Doc/library/doctest.rst

```diff
@@ -277,13 +277,34 @@ Which Docstrings Are Examined?
 The module docstring, and all function, class and method docstrings are
 searched.  Objects imported into the module are not searched.
 
-In addition, if ``M.__test__`` exists and "is true", it must be a dict, and each
+In addition, there are cases when you want tests to be part of a module but not part
+of the help text, which requires that the tests not be included in the docstring.
+Doctest looks for a module-level variable called ``__test__`` and uses it to locate other
+tests. If ``M.__test__`` exists and is ``True``, it must be a dict, and each
```

Comment:
Let's not say "is ``True``" and "must be a dict", it can't be both:

```pycon
>>> __test__ = {
...     'numbers': """
... >>> factorial(6)
... 720
...
... >>> [factorial(n) for n in range(6)]
... [1, 1, 2, 6, 24, 120]
... """
... }
>>> __test__ is True
False
```

The original was "is true" (in quotes) to imply it has a truthy value, rather than literally being `True`.

Can we reword it?